### ISSUE TITLE
Installation fails on CentOS7  - sudo cpanm Zonemaster-Engine-v3.0.3.tar.gz

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-engine#662]
```
cp lib/Zonemaster/Engine/Exception.pm blib/lib/Zonemaster/Engine/Exception.pm
cp lib/Zonemaster/Engine/TestMethods.pm blib/lib/Zonemaster/Engine/TestMethods.pm
: share/Makefile
cd share && make extract-pot touch-po all
make[1]: Entering directory `/root/.cpanm/work/1572519357.1191/Zonemaster-Engine-v3.0.3/share'
touch: missing file operand
Try 'touch --help' for more information.
make[1]: *** [touch-po] Error 1
make[1]: Leaving directory `/root/.cpanm/work/1572519357.1191/Zonemaster-Engine-v3.0.3/share'
make: *** [pm_to_blib] Error 2
-> FAIL Installing Zonemaster-Engine-v3.0.3.tar.gz failed. See /root/.cpanm/work/1572519357.1191/build.log for details. Retry with --force to force install it.
5 distributions installed
```

**Comment by mattias-p** [comment]
I believe #664 will solve this.

**Comment by vlevigneron** [comment]
@sandoche2k Do you have more details ?

```
[levigner@vps430859 test_cpanm]$ cpanm v3.0.3.tar.gz
--> Working on v3.0.3.tar.gz
Fetching file:///home/levigner/Zonemaster/test_cpanm/v3.0.3.tar.gz ... OK
Configuring zonemaster-engine-3.0.3 ... OK
Building and testing Zonemaster-Engine-v3.0.3 ... OK
Successfully installed Zonemaster-Engine-v3.0.3
1 distribution installed
[levigner@vps430859 test_cpanm]$ cat /etc/redhat-release
CentOS Linux release 7.3.1611 (Core)
[levigner@vps430859 test_cpanm]$ perl --version

This is perl 5, version 20, subversion 0 (v5.20.0) built for x86_64-linux

```
It seems to work for me.

**Comment by sandoche2k** [comment]
I have sent you details by secure communication

**Comment by vlevigneron** [comment]
@sandoche2k I confirm that I observe the same issue. But, I also observe that it seems to be fixed by https://github.com/zonemaster/zonemaster-engine/pull/664

**Comment by sandoche2k** [comment]
Issue Fixed by #664
